spec/mailers/bulk_upload_mailer_spec.rb

```diff
@@ -144,6 +139,8 @@
       end
 
       it "sends correctly formed email with A, B, C and more" do
+        error_description = "We noticed the following issues with your upload:\n- logs where the setup sections were incomplete\n- logs with a lot of similar errors in column A, B, C and more\n- logs you are trying to upload have been created previously\n"
+
         expect(notify_client).to receive(:send_email).with(
```

Comment:
Same as above comment - is this expect statement not the final thing to be called in this it block?

Reply:
see other reply